layers/+distributions/spacemacs-base/local/hybrid-mode/hybrid-mode.el

```diff
@@ -118,12 +118,16 @@
   :exit-hook (evil-cleanup-insert-state evil-stop-track-last-insertion)
   :input-method t
   (cond
-   ((evil-hybrid-state-p)
+   ((evil-insert-state-p)
```

Comment:
Should this remain as `evil-hybrid-state-p` since your change is in `hybrid` state definition?